src/odemis/util/angleres.py

```diff
@@ -514,10 +518,26 @@ def Rectangular2Polar(data, output_size, colormap=None):
     # Note: Everything outside of mask is 0
     mask = _CreateMirrorMaskRectangular(data, hole=True)
 
-    # Get the upper/lower intensity limit, by checking for a fixed amount of outliers.
     # In case of NaNs in projection, remove NaNs for calculating histogram limits, but keep for plotting.
     mask_outliers = mask & ~numpy.isnan(data)  # create new mask, which masks NaNs in data
-    lim1, lim2 = img.getOutliers(data[mask_outliers], outliers=1/256)  # 1/256 seems to work well for most datasets
+
+    # define the limits for plotting
+    if data.metadata[MD_POL_MODE] in [MD_POL_UP, MD_POL_DOP, MD_POL_DOLP]:
+        lim1, lim2 = 0, 1
+    # choose limits symmetrically for the following pol pos around 0
+    # DOCP: directly indicates whether data more rhc or lhc polarized
+    elif data.metadata[MD_POL_MODE] in [MD_POL_DOCP, MD_POL_S1N, MD_POL_S2N, MD_POL_S3N,
+                                        MD_POL_DS1N, MD_POL_DS2N, MD_POL_DS3N]:
+        lim1, lim2 = -1, 1
+    else:
+        # Get the upper/lower intensity limit, by checking for a fixed amount of outliers (e.g. cosmic spikes).
+        lim1, lim2 = img.getOutliers(data[mask_outliers], outliers=1/256)  # 1/256 seems to work well for most datasets
+        # choose limits symmetrically for the following pol pos around 0
+        # directly indicates e.g. whether data is more vertically or horizontally polarized
+        if data.metadata[MD_POL_MODE] in [MD_POL_S1, MD_POL_S2, MD_POL_S3,
+                                          MD_POL_DS1, MD_POL_DS2, MD_POL_DS3]:
+            lim = max(abs(lim1), abs(lim2))  # find the max absolute value
```

Comment:
I don't think this comment is useful, it just repeats what you wrote in Python. => drop